### PR TITLE
External links in the footer will now open in newtab

### PR DESCRIPTION
**Pull request by anmode**
# Descripción

Following links in footer will open in newtab
1.All contact us links as these are external not come in site workflow
2.Suppoter logos are already open in new tab I had checked them
3.blog link under site column in footer
4.link under developers section

# Checklist:

- [x] El código sigue la [guía de
      estilo](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) de
      omegaUp.
- [x] Se corrieron todas las pruebas y pasaron.
- [ ] Si se está agregando funcionalidad nueva, se agregaron pruebas.
- [ ] Si el cambio es grande (> 200 líneas), hay que intentar partirlo en
      varios pull requests. De preferencia uno para los controladores + phpunit
      y luego otro para la interfaz.

### CODE REVIEW

**Comment by pabo99** [comment]
LGTM. 

Just one thing, can you apply the change in the following links:
- https://omegaup.org/#about
- https://omegaup.org/#team
?

**Comment by anmode** [comment]
> LGTM.
> 
> Just one thing, can you apply the change in the following links:
> 
> * https://omegaup.org/#about
> * https://omegaup.org/#team
>   ?

But this omegaup org ? And i am working on omegaup.com repo

**Comment by pabo99** [comment]
> > LGTM.
> > Just one thing, can you apply the change in the following links:
> > 
> > * https://omegaup.org/#about
> > * https://omegaup.org/#team
> >   ?
> 
> But this omegaup org ? And i am working on omegaup.com repo but i can do it , there is repo for omegaup org too?

Sorry. I mean, can you add `taget="_blank"`, to those links:
https://github.com/omegaup/omegaup/blob/b3a008fc74b3df0848056c69ee5ffd9d301d6035/frontend/www/js/omegaup/components/common/Footer.vue#L73
https://github.com/omegaup/omegaup/blob/b3a008fc74b3df0848056c69ee5ffd9d301d6035/frontend/www/js/omegaup/components/common/Footer.vue#L78
?

File: frontend/www/js/omegaup/components/common/Footer.vue [hunk 1]
```diff
@@ -72,12 +72,12 @@
           <h4>{{ T.frontPageFooterOrganization }}</h4>
           <ul>
             <li class="mt-1">
-              <a href="https://omegaup.org/#about">{{
+              <a href="https://omegaup.org/#about" target="_blank">{{
```
Comment (pabo99) [inline frontend/www/js/omegaup/components/common/Footer.vue:75]: Thanks!

**Comment by anmode** [comment]
> > > LGTM.
> > > Just one thing, can you apply the change in the following tags:
> > > 
> > > * https://omegaup.org/#about
> > > * https://omegaup.org/#team
> > >   ?
> > 
> > 
> > But this omegaup org ? And i am working on omegaup.com repo but i can do it , there is repo for omegaup org too?
> 
> Sorry. I mean, can you add `taget="_blank"`, to those links:
> 
> https://github.com/omegaup/omegaup/blob/b3a008fc74b3df0848056c69ee5ffd9d301d6035/frontend/www/js/omegaup/components/common/Footer.vue#L73
> 
> 
> https://github.com/omegaup/omegaup/blob/b3a008fc74b3df0848056c69ee5ffd9d301d6035/frontend/www/js/omegaup/components/common/Footer.vue#L78
> 
> 
> ?

done!

**Comment by pabo99** [comment]
LGTM.

Defer to @lhchavez

**Review by pabo99 — APPROVED** [review]

**Pull request merged**